pandas/tests/indexes/interval/test_interval_range.py

```diff
@@ -0,0 +1,301 @@
+from __future__ import division
+
+import pytest
+import numpy as np
+from datetime import timedelta
+from pandas import (
+    Interval, IntervalIndex, Timestamp, Timedelta, DateOffset,
+    interval_range, date_range, timedelta_range)
+from pandas.tseries.offsets import Day
+import pandas.util.testing as tm
+import pandas as pd
+
+
+@pytest.fixture(scope='class', params=['left', 'right', 'both', 'neither'])
+def closed(request):
+    return request.param
+
+
+@pytest.fixture(scope='class', params=[None, 'foo'])
+def name(request):
+    return request.param
+
+
+class TestIntervalRange(object):
```

Comment:
as an aside, we don't generally need classes, unless you have multiple logical groupings within a single file (but then usually we just want separate files). no harm though.